testutils/trace_context.go

```diff
@@ -7,43 +7,48 @@ import (
 // SimpleTraceContextSource is a dummy implementation of TraceContextSource.
 type SimpleTraceContextSource struct{}
 
-// NewRootTraceContext imlements NewRootTraceContext of opentracing.TraceContextSource.
+// NewRootTraceContext implements NewRootTraceContext of opentracing.TraceContextSource.
 func (source *SimpleTraceContextSource) NewRootTraceContext() opentracing.TraceContext {
 	return nil
```

Comment:
we should keep track of the use cases that influence API shape somehow, maybe by (see #xxx) in comments. I don't recall why we needed child tags to be returned from context creation.